package-lock.json

```diff
@@ -10328,22 +10493,14 @@
           }
         },
         "graphql-upload": {
```

Comment:
Weird, this happened in [Ragith's PR](https://github.com/RGPosadas/Mull/pull/200#discussion_r571444083) as well. In my case, running `npm i` reinstalls graphql-upload to 11.0.0.

Reply:
I don't get what the issue is, we have this part in our `package.json`:
```json
 "resolutions": {
    "graphql-upload": "^11.0.0"
  },
```
isn't 11 what we want

Reply:
Yes exactly, 11 is what we want. But your PR brings it back down to 8.1.0

Reply:
oh whoops I misunderstood, I did another npm i and I put it back to 11, I'll be pushing that